pylutron/__init__.py

```diff
@@ -248,6 +248,7 @@ def _parse_area(self, area_xml):
         if device_xml.tag != 'Device':
           continue
         if device_xml.get('DeviceType') in (
+            'HWI_SEETOUCH_KEYPAD'
```

Comment:
need a comma at the end there. You must have not tested this :)

Reply:
Oops.  I actually have the code running on my Home Assistant but when I copied that line of code here to submit it I did miss the comma. Thank you. I’m not a coder. Also, this is the first time I have ever posted something to github  

This line of code opened up access to those devices on my system. People upgrading Lurton Home Works processors to the QS from the legacy like I did will still lave legacy keypads on the system. 

I resubmitted this. Hope I did it properly.